app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/templates/custom_language_selection.scala.html

```diff
@@ -16,18 +16,19 @@
 
 @(langMap: Map[String, Lang], langToCall: String => Call, customClass: Option[String] = None, appName: Option[String] = None)(implicit messages: Messages)
 
-<p class="@if(customClass.isDefined) {@customClass.get}">
-
-@langMap.map { case (key: String, value: Lang) =>
-    @if(messages.lang.code != value.code) {
-        <a href="@langToCall(key)" id="@{key}-switch" data-journey-click="click : language : @value.code" >
-            @key.capitalize
-        </a>
-    } else {
-        @key.capitalize
-    }
-    @if(key != langMap.last._1) {
-        @Html(" | ")
-    }
-}
-</p>
+<nav class="hmrc-language-select" aria-label="Language switcher">
```

Comment:
```suggestion
<nav class="hmrc-language-select" aria-label="@messages("language-selector.nav-label")">
```
Currently it will display to both English and Welsh users as "Language switcher", use a message and add in a welsh variant for people reading in welsh